src/game_initialization/multiplayer.cpp

```diff
@@ -682,8 +682,8 @@ void start_local_game_commandline(CVideo& video, const config& game_config, save
 	}
 
 	std::string label = "";
```

Comment:
Honestly, I think it was just used in the commented-out line below it... the `resources::recorder->add_log_data("ai_log","ai_label",label);` one.